test/TestAlgorithms.js

```diff
@@ -0,0 +1,19 @@
+
+const TestBuffer = artifacts.require("TestBuffer");
```

Comment:
This file should probably be called something else? And we don't really need the foreach.

Reply:
What else would you call?